ui/test/browser/index.html

```diff
@@ -22,7 +24,8 @@
 
 <body>
   <div id="app"></div>
-  <script src="https://unpkg.com/mocha/mocha.js"></script>
+  <script src="mocha.js"></script>
+  <!-- script src="https://unpkg.com/mocha/mocha.js"></script -->
```

Comment:
This is the worst part of this, I don't like that you have to explicitly copy mocha.js for local development, or otherwise have to pay attention to this file. TODO: DON'T FOR GET

Will try to fix before merging in

Reply:
(Fixed it)